includes/definitions/horizon.yaml

```diff
@@ -0,0 +1,10 @@
+os: horizon
+text: 'Dragonwave Horizon Compact'
+type: wireless
+icon: dragonwave
+over:
+    - { graph: device_wireless_snr, text: 'SNR' }
+    - { graph: device_wireless_power, text: 'Tx Power' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.7262.2.2
```

Comment:
Looks like you can change that to just `1.3.6.1.4.1.7262.2.` to pick up more Horizon devices.